src/main/scala/is/hail/utils/richUtils/RichIndexedRowMatrix.scala

```diff
@@ -40,18 +39,18 @@ object RichIndexedRowMatrix {
   }
 }
 
-class RichIndexedRowMatrix(indexedRowMatrix: IndexedRowMatrix) {
+class RichIndexedRowMatrix(irm: IndexedRowMatrix) {
```

Comment:
Same.  I often review the differences before I PR and remove inessential changes that got made during the development of the PR.